### ISSUE TITLE
On Win10, it restarts over and over in the middle of the boot process.

### ISSUE THREAD

**Issue by hidao80** [KEINOS/Vagrantfile_for_macOS#3]
My environment:
- Host:
    - Windows 10 Home 64bit
        - Processer: Intel Core i7-7500U CPU @ 2.70GHz 2.90GHz
        - RAM: 16GB
        - laptop
    - Virtualbox: 5.2.26
    - Vagrant: 2.2.0
- Guest:
    - KEINOS/macOS.10.13.6_Japanese ver. 0.0.1
    - GUI: true
    - VRAM:  1024

Vagrantfile:  
```rb
Vagrant.configure("2") do |config|
  config.vm.box = "KEINOS/macOS.10.13.6_Japanese"
  config.vm.box_version = "0.0.1"
  config.vm.provider "virtualbox" do |vb|
    vb.gui = true
    vb.memory = "1024"
  end
end
```

Do you mistaken how to write a Vagrant file?
Is there anything I can do?

Thanks.

**Comment by hidao80** [comment]
I was able to start macOS in my environment!
It is doubtful whether this method is good ...

https://techsviewer.com/install-macos-high-sierra-virtualbox-windows/

**Comment by KEINOS** [comment]
@hidao80 

Glad to hear that!

In which step of the article took effect to boot up on Win?
Will you try with the below settings in your free time?

```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "KEINOS/macOS.10.13.6_Japanese"
  config.vm.box_version = "0.0.1"
  config.vm.synced_folder ".", "/vagrant", disabled: true
  config.vm.provider "virtualbox" do |vb|
    vb.gui = true
    vb.memory = "2048"
  end
end
```
```diff
+ config.vm.synced_folder ".", "/vagrant", disabled: true
-     vb.memory = "1024"
+     vb.memory = "2048"
```

High Sierra needs more than 2 GB of memory and lack of memory causes rebooting.
And also you need to disable the `synced_folder`.

If it works, then these settings are fixed in the [box version 0.0.2 released](https://app.vagrantup.com/KEINOS/boxes/macOS.10.13.6_Japanese/versions/0.0.2).
So it should work as below:

```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "KEINOS/macOS.10.13.6_Japanese"
  config.vm.box_version = "0.0.2"
  config.vm.provider "virtualbox" do |vb|
    vb.gui = true
  end
end
```
```diff
-  config.vm.box_version = "0.0.1"
+  config.vm.box_version = "0.0.2"
-  config.vm.synced_folder ".", "/vagrant", disabled: true
-     vb.memory = "2048"
```

**Comment by KEINOS** [comment]
Otherwise, the command below in the article might be the key.

```PowerShell
> cd "C:\Program Files\Oracle\VirtualBox\"
> VBoxManage.exe modifyvm "Your VM Name" --cpuidset 00000001 000106e5 00100800 0098e3fd bfebfbff
> VBoxManage setextradata "Your VM Name" "VBoxInternal/Devices/efi/0/Config/DmiSystemProduct" "iMac11,3"
> VBoxManage setextradata "Your VM Name" "VBoxInternal/Devices/efi/0/Config/DmiSystemVersion" "1.0"
> VBoxManage setextradata "Your VM Name" "VBoxInternal/Devices/efi/0/Config/DmiBoardProduct" "Iloveapple"
> VBoxManage setextradata "Your VM Name" "VBoxInternal/Devices/smc/0/Config/DeviceKey" "ourhardworkbythesewordsguardedpleasedontsteal(c)AppleComputerInc"
> VBoxManage setextradata "Your VM Name" "VBoxInternal/Devices/smc/0/Config/GetKeyFromRealSMC" 1
```

I will check my current VM setting and see the differences.
So we can add them to the default Vagrantfile. =)

Again, thanks for the information!

**Comment by KEINOS** [comment]
Seems that the low memory and not disabling `synced_folder` **were NOT the cause** ...

I tried with the setting below and even giving an error the VM launched.

```ruby
Vagrant.configure("2") do |config|
  config.vm.box = "KEINOS/macOS.10.13.6_Japanese"
  config.vm.box_version = "0.0.1"
  config.vm.provider "virtualbox" do |vb|
    vb.gui = true
    vb.memory = "1024"
  end
end
```
<kbd>![virtualbox_highsierra_default_1551923586556_59571_07_03_2019_10_59_22](https://user-images.githubusercontent.com/11840938/53927022-da342c80-40c8-11e9-9ad3-97bc1c01383e.png)</kbd>

So adding the CPU ID set, EFI and SMC config to the default Vagrant file might be needed.

**Comment by KEINOS** [comment]
@hidao80 

I updated (v0.0.3) the fixed base box. Which includes the machine settings such as CPU IDs and etc.

- [KEINOS/macOS.10.13.6_Japanese v0.0.3](https://app.vagrantup.com/KEINOS/boxes/macOS.10.13.6_Japanese/versions/0.0.3) @ VagrantCloud

Here's the Vagrantfile applied to the current version (v0.0.3) as a refference.
- [Default Vagrantfile for KEINOS/macOS.10.13.6_Japanese v0.0.3](https://github.com/KEINOS/Vagrantfile_for_macOS/blob/master/base-boxes/macOS.10.13.6_Japanese/Vagrantfile_v0.0.3) @ GitHub

Hope with this box it launches HighSierra Guest straight forward in Windows10 Host.

**Comment by KEINOS** [comment]
macOS HighSierra and Mojave Base boxes updated! Fixed PR #4.

- [KEINOS/macOS.10.13.6_Japanese v0.0.4](https://app.vagrantup.com/KEINOS/boxes/macOS.10.13.6_Japanese/versions/0.0.4) @ VagrantCloud
- [KEINOS/macOS.10.14.1_Japanese v0.0.3](https://app.vagrantup.com/KEINOS/boxes/macOS.10.14.1_Japanese/versions/0.0.3) @ VagrangCloud

**Comment by hidao80** [comment]
Operation report:

By using [Vagrantfile_v0.0.4](https://github.com/KEINOS/Vagrantfile_for_macOS/blob/master/base-boxes/macOS.10.13.6_Japanese/Vagrantfile_v0.0.4), HighSierra was launched with a single "vagrant up" command!

I did it!
